pkg/api/unversioned/time.go

```diff
@@ -76,6 +76,12 @@ func (t Time) Rfc3339Copy() Time {
 	return Time{copied}
```

Comment:
It's not being used by anyone, but I feel it would be useful if someone want to get second-level copies. Your call :)